### ISSUE TITLE
ruby -n option is not implemented

### ISSUE THREAD

**Issue by deepj** [oracle/truffleruby#1532]
As the title says. This option is mentioned in help (`ruby --help`). So I would expect it works in the recent version of TruffleRuby :)

**Comment by faraazahmad** [comment]
I would like to take up this issue

**Comment by chrisseaton** [comment]
Great! It should be pretty easy.

Please check our [contributing document](https://github.com/oracle/truffleruby/blob/master/CONTRIBUTING.md) before spending time working on a fix.

**Comment by eregon** [comment]
Here is the extract from the man page:
```
     -n             Causes Ruby to assume the following loop around your script, which makes it iterate over file name arguments some‐
                    what like sed -n or awk.

                          while gets
                            ...
                          end
```

**Comment by bjfish** [comment]
This is now implemented at 4a944fc12b78a91a9d784697e11be011d2e85e50.
